test/helpers.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var expect = require('must');
+var STANDARD_DELAY = 200; // 200ms between nested calls to the API so things settle
```

Comment:
If we add this delay I feel we aren't testing the way our users will employ the library. Even in my projects I run the next async call as soon as the callback returns. Any comment @aendrew?